packages/lab/stories/multiline-input/multiline-input.stories.tsx

```diff
@@ -100,6 +100,55 @@ export const Disabled: ComponentStory<typeof MultilineInput> = (args) => {
     </FlowLayout>
   );
 };
+
+export const CharacterCount: ComponentStory<typeof MultilineInput> = (args) => {
+  const { announce } = useAriaAnnouncer();
+  const [value, setValue] = useState<string>("Value");
+
+  const ANNOUNCER_DELAY = 100;
+  const MAX_CHARS = 10;
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const newVal = event.target.value;
+
+    if (newVal.length < MAX_CHARS) {
+      const value = newVal;
+      setValue(value);
+    }
+
+    if (MAX_CHARS - newVal.length === 50) {
```

Comment:
Has this announcer logic been tested? Additions like that can actually harm accessibility rather than improve it.

Also as it is currently written this logic is broken. You can only write 9 characters and the only announcements you can achieve are:

10 characters remaining and once at the maximum value you can type (9 characters) you can press another key to then have it has 0 characters remaining but nothing gets updated/

Reply:
doh, I did test it, but then changed it to 10 characters as thought it would be a clearer demo then didnt update the announcer to match 🤣 
I've deleted it in the hope to get this out asap, we can add something back if recommended by stella in the future